com.unity.UnityHub.yaml

```diff
@@ -0,0 +1,84 @@
+app-id: com.unity.UnityHub
+branch: stable
+base: io.atom.electron.BaseApp
+base-version: '18.08'
+runtime: org.freedesktop.Platform
+runtime-version: '18.08'
+sdk: org.freedesktop.Sdk
+command: start-unityhub
+separate-locales: false
+tags: [proprietary]
+finish-args:
+  - '--allow=multiarch'
```

Comment:
No need for quotes here, unless you prefer them. Same goes for the cleanups and build-commands, but really it’s your choice.

Reply:
I tend to prefer them, just because I know YAML can sometimes interpret things oddly (though flatpak-builder is relatively liberal with what it interprets as a string).